testing/PIMS.Tests.Automation/Classes/AcquisitionFile.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace PIMS.Tests.Automation.Classes
+﻿using System.Drawing;
+
```

Comment:
I usually just approve these PRs but I have a question - do you really need **System.Drawing** here?
It seems like this dependency might be added by accident

Reply:
No, will delete it for the next PR. Thanks for catching that up